### DISCUSSION TITLE
Date and datetime formats

### DISCUSSION THREAD

**Discussion by sandron13** [h2oai/datatable#2693] [General]
Hello everyone!
First of all want to say a BIG thank you to all of you for this incredible project & I very hope that it still be developed in future. But I have some questions how to handle with date and datetime formats, Is it possible in data.table ? If yes could you please show me some examples because i haven`t found any of it :(

**Comment by st-pasha** [comment]
The datetime functionality has not been implemented yet, but we hope to start working on it soon.

**Reply by st-pasha** [reply]
We now have date and time functions, since release 1.0.0
